### PR TITLE
DEP: remove the erf() and erfc function fixes #1778

### PR DESCRIPTION
**Pull request by ShunyiYang**
This time, I used GitHub Desktop. Previously, I used the Git command line, but there was a line ending issue: I am on a Windows system, and after downloading cogent3 to my computer, all the files had \r\n line endings instead of \n, which resulted in over a million modifications when compared to the original repository. This problem doesn't occur with GitHub Desktop. However, I still want to know how to solve this issue if using the Git command line.

### CODE REVIEW

**Comment by ShunyiYang** [comment]
Oh, I get it now. After changing the VSCode settings, I need to redownload cogent3 so that VSCode can open files in LF mode.

File: src/cogent3/maths/stats/special.py [hunk 1]
```diff
@@ -295,6 +297,8 @@ def erfc(a):
         return y
 
 
+
```
Comment (GavinHuttley) [inline src/cogent3/maths/stats/special.py:300]: you should run black and isort against the code. In the root directory for the project
```
$ black .; isort .
```
these ensure formatting, white space etc.. all match the project standards

**Review by GavinHuttley — CHANGES_REQUESTED** [review]
Thanks so much for the contribution!

One small task left before wrapping up your first time contribution to Cogent3 🚀! Run the linters against the code (see comment on the file).

**Comment by GavinHuttley** [comment]
fixes #1738

**Pull request merged**